src/bin/add/main.rs

```diff
@@ -72,8 +73,8 @@ dependencies (version set to "*").
 "#;
 
 fn handle_add(args: &Args) -> Result<(), Box<Error>> {
-    let mut manifest = try!(Manifest::open(&args.flag_manifest_path.as_ref().map(|s| &s[..])));
-    let deps = try!(args.parse_dependencies());
+    let mut manifest = Manifest::open(&args.flag_manifest_path.as_ref().map(|s| &s[..]))?;
```

Comment:
Memo to self: Hm, I think I know a better way to get this working by now, `.as_ref().map(|s| &s[..])` looks quite ugly.